### PR TITLE
Update README on howto deploy custom plans

### PR DESCRIPTION
**Pull request by mjura**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,15 +8,22 @@ A plan is just a container image which is executed in the host context without a
...
+
+```bash
+docker build -t elemental-example-plan .
 ```
 
 and then push the image to a registry, as it should be accessible remotely by nodes.
 
-Plans can be either specified remotely via a management cluster, or locally by writing plan files in `/var/lib/elemental/agent/plans` inside a node host folder, which is running the `rancher-system-agent`. 
+Plans can be either specified remotely via a management cluster, or locally by writing plan files in `/var/lib/elemental/agent/plans` inside a node host folder, which is running the `rancher-system-agent`.
 Note: Elemental configures the `/var/lib/elemental` path over the standard `/var/lib/rancher` path.
```
Comment (fgiudici) [inline README.md]: This is true only till the cluster provisioning is triggered on: at that point, a new rancher-system-agent is downloaded and will take over the old instance, moving the configuration files to /var/lib/rancher. The plan for the (hopefully near)  future is to just have an instance and keep the same config for the whole host lifetime... 
so, if you wan to run a local plan *after* cluster provisioning, you need to switch to the `/var/lib/rancher` path.

File: README.md [hunk 2]
```diff
@@ -40,15 +47,43 @@ stages:
          group: 0
 ```
 
+When you want to register node in Rancher Manager and create local plan from there:
```
Comment (fgiudici) [inline README.md]: I would link this to the elemental-operator readme... something like:
```suggestion
You can create a local plan via a [MachineRegistration CRD](https://github.com/rancher/elemental-operator#machineregistration) , used to register a node in Rancher Manager
```
but it is just a nit, feel free to ignore this!

**Review by fgiudici — APPROVED** [review]
lgtm!

File: README.md [hunk 3]
```diff
@@ -40,15 +47,43 @@ stages:
...
+      - name: root
+        passwd: linux
+      write_files:
+      - path: /var/lib/elemental/agent/plans/test.plan
+        permissions: "0600"
+        content: |
+          {"instructions":[{"name":"test","image":"ghcr.io/rancher-sandbox/elemental-example-plan:8h"}]}
+    elemental:
+      install:
+        automatic: true
```
Comment (fgiudici) [inline README.md]: Automatic was ignored and is no more in the elemental-operator code
```suggestion

```

File: README.md [hunk 4]
```diff
@@ -40,15 +47,42 @@ stages:
...
+  config:
+    cloud-config:
+      users:
+      - name: root
+        passwd: linux
+      write_files:
+      - path: /var/lib/elemental/agent/plans/test.plan
+        permissions: "0600"
+        content: |
+          {"instructions":[{"name":"test","image":"ghcr.io/rancher-sandbox/elemental-example-plan:8h"}]}
```
Comment (Itxaka) [inline README.md]: awesome!

**Review by Itxaka — APPROVED** [review]
awesome, very useful!

**Pull request merged**